targets/mbedos5/template-mbedignore.txt

```diff
@@ -1,6 +1,7 @@
 cmake/*
 docs/*
 jerry-libc/*
+jerry-libm/include/*
 jerry-main/*
```

Comment:
Why not exclude the whole jerry-libm directory and let the libm of the toolchain be used? If the header comes from the toolchaim but the sources come from jerry-libm, nasty confusion can happen. (Not tried myself but that's my gut feeling.)

Reply:
@akosthekiss Good point, thanks. Have updated the PR and verified that it still builds.

Reply:
I don't see the new changes in the PR, yet. Problem on my side?